modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in all namespaces. After the objects are removed, you can then remove the Operator and its namespace.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
```

Comment:
```suggestion
* You have access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
```
I know this went through peer review but I really think this should be fixed to be a complete sentence to have parallel structure with the second bullet item. Sorry :(